### PR TITLE
Gmail Login Page

### PR DESCRIPTION
**Pull request by AKD-2024**
Create index.html, style.css, and assets folder for google logo and update README.md file.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+.main {
+  padding: 1.5rem;
+}
+
+img {
+  height: 3rem;
+  width: 3rem;
+}
+
```
Comment (PurnimaCompro) [inline style.css:17]: css should not be applied on selector

**Review by PurnimaCompro — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          </select>
+          <ul>
+           <li>Help</li>
+           <li>Privacy</li>
+           <li>Terms</li>
+          </ul> 
+      </footer>
+      </div>
+    </body>
+</html>
```
[the remark this answers is not in the document]
Comment (AKD-2024) [reply index.html:47]: @PurnimaCompro Ma'am, what is the git error?

File: style.css (hunk 1 above)
Comment (AKD-2024) [reply style.css:17]: @PurnimaCompro I think you meant directly on the tag, I know but only single img tag was used so it was not going to affect anywhere that is why I did this. Although resolved!

File: style.css [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+#email {
+  border: 1px solid #444746;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  height: 1.7rem;
+  margin: 1.4rem auto 0.7rem auto;
+  padding: 0.8rem 1rem;
+  width: 90%;
+}
```
[the remark this answers is not in the document]
Comment (AKD-2024) [reply style.css:38]: @PurnimaCompro Ma'am, there are three selectors in CSS, I can't apply styles to id and tags. So I am left with class only.  Also resolved!

**Review by PurnimaCompro — APPROVED** [review]

**Pull request merged**